test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -11163,6 +11158,20 @@ def revert_to_defaults(self):
         self.resource_group = None
         return self.isUp()
 
+    def revert_server_attribs(self):
+        """
+        Method to unset only server attributes during tearDown
+        """
+        unset_list = ['managers', 'operators']
+        self.manager(MGR_CMD_UNSET, SERVER, unset_list, sudo=True)
+
+        current_user = pwd.getpwuid(os.getuid())[0]
+        a = {ATTR_managers: (INCR, current_user + '@*')}
```

Comment:
I don't know if it is worthwhile, but if the user is already in managers, this spits out an error.  It's not a harmful error though.  We want the user in managers.  We could get rid of this error by checking first, but it will require a stat call to the server.  Is it worth it to avoid printing an error?

Reply:
@bhroam I am unsetting managers attribute before I set it to current user. I am doing this because it might be set to some other users in addition to current user as part of few test cases. I am also handling error which will be likely thrown from "self.manager(MGR_CMD_SET, SERVER, a, sudo=True)"

Reply:
Great, thanks